### PR TITLE
Create new proposal for AI cough analysis program

### PR DESCRIPTION
**Pull request by djmoffat**

### CODE REVIEW

File: software/proposals/cough_analysis.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+title: "Cough Analysis"
+---
+
+| | |
+|-|-|
+| Proposal Stage |   Draft   |
+| Tags           |   Audio, Analysis, Diagnosis, Cough   |
+| Created Date   |   18/03/2020   |
+| Lead           |   Dave Moffat   |
+| Slack Channel  |   proj-dev-ai-cough-detection   |
```
Comment (mazaira) [inline software/proposals/cough_analysis.md:12]: I couldn't find this channel, is it valid?

Comment (djmoffat) [reply software/proposals/cough_analysis.md:12]: Yes, this is the channel. It is on the open-covid19 Slack project

Comment (mazaira) [reply software/proposals/cough_analysis.md:12]: are we talking about the same Slack organization? helpfulengineering.slack.com ?
I couldn't find any. You can ping me privately on Slack if you prefer to sort it out

Comment (djmoffat) [reply software/proposals/cough_analysis.md:12]: No, we are not. I am talking about open-covid19.slack.com

Comment (mazaira) [reply software/proposals/cough_analysis.md:12]: This repo belongs to the HelpfulEngineering one. I think you pushed it to the wrong place 🙂
What is the goal with this PR? Is the project a work in progress thing that you are managing separately?
If you want to migrate it, feel invited to join our Slack, get some volunteers around the idea, etc

**Comment by mazaira** [comment]
Closing by lack of feedback. Please feel free to re-open if there would be any changes from the current state.

**Pull request closed without merging**